### PR TITLE
propka package added

### PR DESCRIPTION
**Pull request by tsenapathi**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/propka/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+test:
+  commands:
+    - propka31 -h
+
+about:
+    home: https://github.com/jensengroup/propka-3.1.git
+    dev_url: https://github.com/jensengroup/propka-3.1.git
+    license: GNU Lesser General Public License v2.1
+    license_family: GPL
+    license_file: LICENSE.txt
```
Comment (bgruening) [inline recipes/propka/meta.yaml]: if the license file exists upstream like here [LICENSE](https://github.com/jensengroup/propka-3.1) you don't need to include it.

File: recipes/propka/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - propka31 -h
+
+about:
+    home: https://github.com/jensengroup/propka-3.1.git
```
Comment (bgruening) [inline recipes/propka/meta.yaml]:
```suggestion
    home: https://github.com/jensengroup/propka-3.1
```

File: recipes/propka/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    dev_url: https://github.com/jensengroup/propka-3.1.git
+    license: GNU Lesser General Public License v2.1
```
Comment (bgruening) [inline recipes/propka/meta.yaml]: Simply `GPL 2.1`

**Comment by tsenapathi** [comment]
@bgruening Fixed the license issue but getting a error,
```
[noarch_python] Noarch package contains binary script: propka31.exe
Command exited with code 1
```
Do you know how to fix this?

File: recipes/propka/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    - python
+
+test:
+  commands:
+    - propka31 -h
+
+about:
+    home: https://github.com/jensengroup/propka-3.1
+    license: LGPL
+    license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (bgruening) [inline recipes/propka/meta.yaml]: Please read the documentation about this section.

LICENSE can be taken out of the upstream tarball.

Please also use `conda skeleton pypi propka` to generate those recipes. You will see you get a much better one.

**Comment by tsenapathi** [comment]
@bgruening Thank you!!
It worked !!

File: recipes/propka/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - setuptools
+
+test:
+  commands:
```
Comment (bgruening) [inline recipes/propka/meta.yaml]: the skeleton has also generated one test more ...isn't it?

File: recipes/propka/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    - propka31 -h
+
+about:
+  home: http://propka.org
+  license: GNU Lesser General Public v2 (LGPLv2)
+  license_family: LGPL
+  license_file:
```
Comment (bgruening) [inline recipes/propka/meta.yaml]:
```suggestion
  license_file: LICENSE
```

File: recipes/propka/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  commands:
+    - propka31 -h
+
+about:
+  home: http://propka.org
+  license: GNU Lesser General Public v2 (LGPLv2)
+  license_family: LGPL
+  license_file: 
+  summary: Heuristic pKa calculations with ligands
+  doc_url:
```
Comment (bgruening) [inline recipes/propka/meta.yaml]: remove doc_url if not used

File: recipes/propka/meta.yaml (hunk 5 above)
Comment (tsenapathi) [reply recipes/propka/meta.yaml]: Added!

File: recipes/propka/meta.yaml (hunk 6 above)
Comment (tsenapathi) [reply recipes/propka/meta.yaml]: Should I add the LICENSE file?

File: recipes/propka/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    - propka31 = propka.run:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/propka/meta.yaml]: is that really needed as runtime dependency?

Comment (tsenapathi) [reply recipes/propka/meta.yaml]: Yes, I get an error If i do not include it

**Comment by tsenapathi** [comment]
@bgruening the error is  `ModuleNotFoundError: No module named 'pkg_resources'`
This related to setuptools

**Pull request merged**